Backpack/CardButton/Classes/CardButtonViewConfigurator.swift

```diff
@@ -0,0 +1,50 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Hold shared logic between BPKSaveCardButton & BPKShareCardButton
+final class CardButtonViewConfigurator {
+
+    // CardButton Required a larger hit area of at least 48px
+    let widthHeight = BPKSpacingLg * 2
+
+    func createContainedBackgroundCircle() -> UIView {
+        let circle = UIView()
+        circle.backgroundColor = BPKColor.textDisabledOnDarkColor.withAlphaComponent(0.5)
```

Comment:
This color already has an alpha of 0.5.
Can you remove the `.withAlphaComponent(0.5)` please

Reply:
good catch! thanks for finding this

Reply:
I changed it in a0ff090c03c3c2d61f30b2c482ae91689dd5a93b